### DISCUSSION TITLE
I want to use qemu to boot  redhat6.0 with OVMF.fd, it not work well

### DISCUSSION THREAD

**Discussion by yangni2017** [tianocore/edk2#4365] [Q&A]
### Summary
I want to boot  redhat6.0 with OVMF.fd, it not work well,the environment is very tight,use vnc to connect,The screen always is stuck and cannot be operated

### Details
**vm info:**  
version: redhat6.0    2.6.32(kernel)

**host info:**
version: centos8   4.19(kernel)
qemu: 6.2.0

**edk2:**
branch:  UDK2018
suport  X64 & IA32
compile:
make -C BaseTools/Source/C
./OvmfPkg/build.sh -D DEBUG_ON_SERIAL_PORT

**Comment by ghost** [comment]
Could you try building it again with `-D DEBUG_ON_SERIAL_PORT=TRUE`?
I think the =TRUE is needed.

**Reply by yangni2017** [reply]
I do it as you say. bug it seem not work, I use vnc connect qemu, it is also a black screen 
![image](https://github.com/tianocore/edk2/assets/24664071/20a16306-8fe8-4450-a61b-cc3696e6aef4)

**Reply by yangni2017** [reply]
I want to know which edk2 version can supprt redhat6.0 ?

**Comment by yangni2017** [comment]
Can someone help me?
